tests/integration/expected/rust_sdk_account_test/miden_sdk_account_test.hir

```diff
@@ -1,8 +1,8 @@
 (component 
     ;; Component Imports
-    (lower (import #add_asset<0x0000000000000000000000000000000000000000000000000000000000000000> (digest 0x0000000000000000000000000000000000000000000000000000000000000000) (type fn(i32, i32, i32, i32) -> i32, i32, i32, i32)) (#miden:tx_kernel/account #add_asset<0x0000000000000000000000000000000000000000000000000000000000000000>)
-    (lower (import #get_id<0x0000000000000000000000000000000000000000000000000000000000000000> (digest 0x0000000000000000000000000000000000000000000000000000000000000000) (type fn() -> i32)) (#miden:tx_kernel/account #get_id<0x0000000000000000000000000000000000000000000000000000000000000000>)
-    (lower (import #get_inputs<0x0000000000000000000000000000000000000000000000000000000000000000> (digest 0x0000000000000000000000000000000000000000000000000000000000000000) (type fn(i32) -> i32, i32)) (#miden:tx_kernel/note #get_inputs<0x0000000000000000000000000000000000000000000000000000000000000000>)
+    (lower (import_miden () (digest 0x0000000000000000000000000000000000000000000000000000000000000000) (type fn(i32, i32, i32, i32) -> i32, i32, i32, i32)) (#miden:tx_kernel/account #add_asset<0x0000000000000000000000000000000000000000000000000000000000000000>)
```

Comment:
Looks like I missed two things when porting over the textual format:

1. The interface function type is not formatted using the proper type syntax, it should be `(type (func (param i32) (param i32) .. (result i32))`
2. If the digest is in the imported name, we probably don't need to duplicate it as a separate attribute/field of the import, and instead just expect to parse it out of the name (or vice versa, drop it from the name and expect to get it from the `digest` attribute)

Anyway, heads up on that, we'll need to fix that to ensure the text format is round-trippable

Reply:
1. I missed this. Done. 
2. Good catch! Yep, I already parsing it only once and storing it in `digest`. So I removed the digest from the function name in https://github.com/0xPolygonMiden/compiler/pull/140/commits/9176975dfc8b8b048f9d5cc28228bc0882b34951